pypeit/scripts/coadd_datacube.py

```diff
@@ -143,10 +148,17 @@ def coadd_cube(files, parset, overwrite=False):
         rel_humidity = spec.get_meta_value([spec2DObj.head0], 'humidity')
         coord = SkyCoord(raval, decval, unit=(units.deg, units.deg))
         location = spec.location  # TODO :: spec.location should probably end up in the TelescopePar (spec.telescope.location)
-        ra_corr, dec_corr = dc_utils.dar_correction(waveimg[onslit_gpm], coord, obstime, location,
-                                                    pressure, temperature, rel_humidity, wave_ref=wave_ref)
-        raimg[onslit_gpm] += ra_corr
-        decimg[onslit_gpm] += dec_corr
+        if pressure.value == 0.0:
```

Comment:
Again, beware of astropy quantities, they are really bad news in terms of performance. Fine as long as you don't accidentally convert an array into a Quantity.

Reply:
See earlier comment - this is now reverted back to float.